evap/results/views.py

```diff
@@ -27,24 +32,55 @@ def delete_template_cache(evaluation):
 
 
 def _delete_template_cache_impl(evaluation):
+    _delete_evaluation_template_cache_impl(evaluation)
+    if evaluation.course.evaluations.count() > 1:
```

Comment:
why only delete the course template cache when there are more evaluations?

Reply:
the thought was that there is none if the course has only one evaluation but probably just trying to delete it anyway although it might not exist is faster so we don't need to count that first.